drivers/adc/adc_ad4114.c

```diff
@@ -0,0 +1,457 @@
+/*
+ * Copyright (c) 2024 Pierrick Curt
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdbool.h>
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/adc.h>
+#include <zephyr/drivers/spi.h>
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/util.h>
+
+#define ADC_CONTEXT_USES_KERNEL_TIMER
+#include "adc_context.h"
+
+LOG_MODULE_REGISTER(ADC_AD4114, CONFIG_ADC_LOG_LEVEL);
+
+#define DT_DRV_COMPAT adi_ad4114_adc
+
+#define AD4114_ADC_VREF_MV    2500U
```

Comment:
This value does not seem to be used anywhere.

Reply:
Yes, I removed it